solutions/project/hub-env/services.yaml

```diff
@@ -0,0 +1,40 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Compute API
+apiVersion: serviceusage.cnrm.cloud.google.com/v1beta1
+kind: Service
+metadata:
+  name: hub-project-id-compute # kpt-set: ${hub-project-id}-compute
+  namespace: projects
+  annotations:
+    config.kubernetes.io/depends-on: resourcemanager.cnrm.cloud.google.com/namespaces/projects/Project/hub-project-id # kpt-set: resourcemanager.cnrm.cloud.google.com/namespaces/projects/Project/${hub-project-id}
+    cnrm.cloud.google.com/deletion-policy: "abandon"
+    cnrm.cloud.google.com/disable-dependent-services: "false"
+    cnrm.cloud.google.com/project-id: hub-project-id # kpt-set: ${hub-project-id}
+spec:
+  resourceID: compute.googleapis.com
+---
+# DNS API
+apiVersion: serviceusage.cnrm.cloud.google.com/v1beta1
+kind: Service
+metadata:
+  name: hub-project-id-dns # kpt-set: ${hub-project-id}-dns
+  namespace: projects
+  annotations:
+    config.kubernetes.io/depends-on: resourcemanager.cnrm.cloud.google.com/namespaces/projects/Project/hub-project-id # kpt-set: resourcemanager.cnrm.cloud.google.com/namespaces/projects/Project/${hub-project-id}
+    cnrm.cloud.google.com/disable-on-destroy: "false"
```

Comment:
just curious if you noticed a difference between [`cnrm.cloud.google.com/disable-on-destroy: "false"`](https://cloud.google.com/config-connector/docs/reference/resource-docs/serviceusage/service) and [`cnrm.cloud.google.com/deletion-policy: "abandon"`](https://cloud.google.com/config-connector/docs/reference/annotations) ?

Reply:
to be honest, I haven't investigated that annotation very much. Anyone else has done some testing about this ?